### DISCUSSION TITLE
Customer has rights to asin for AYCL

### DISCUSSION THREAD

**Discussion by mynameisdirk** [openaudible/openaudible#1043] [Q&A] [answered]
Yesterday I downloaded some new audiobooks from Audible - which were kind of free books. As I converted them to a wrong bitrate I wanted to re-download them, but the message above appeared.

So I was wondering, why it worked yesterday once and why it is not working today anymore? A change of rights from one day to the next?

**Comment by openaudible** [comment, marked as answer]
The free titles have expiration dates where they become no longer available in your library. You may need to go to your audible account and make sure the titles are still available. A full library resync from the control menu may also help.
